C# Examples/EposServiceConsumer/UnitTests/App.config.local

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <appSettings>
+    <add key="ConsumerKey" value="ITG4ck3Q6d6lB43ytvfwx9GFUUG6eSqg"/>
+    <add key="ConsumerSecret" value="u4Cg6LXCgVx4yLJmqGEryyO1GEaXNFGL"/>
```

Comment:
I don't think it's a good idea to leave these credentials in - even if they're just from someone's local setup. It'd be too easy to accidentally leak actual credentials. I'd just leave them empty.